wiki/items/armor.md

```diff
@@ -1,5 +1,178 @@
+
+---
+
+Next, we register the armor in our mod initializer and add it to the combat item group. There are two things to notice about this:
+
+1. We name the armor items the material name combined with their variant.
+2. We add them to the item group using the `addAfter` method. Because it is an armor, we want to have it appear next to all the vanilla armors. So we use that method, which can add a variable amount of items after another specific, already present item.
+
+```java
+Registry.register(Registries.ITEM, new Identifier(mod.metadata().id(), "example_item_helmet"), EXAMPLE_HELMET);
+Registry.register(Registries.ITEM, new Identifier(mod.metadata().id(), "example_item_chestplate"), EXAMPLE_CHESTPLATE);
+Registry.register(Registries.ITEM, new Identifier(mod.metadata().id(), "example_item_leggings"), EXAMPLE_LEGGINGS);
+Registry.register(Registries.ITEM, new Identifier(mod.metadata().id(), "example_item_boots"), EXAMPLE_BOOTS);
+ItemGroupEvents.modifyEntriesEvent(ItemGroups.COMBAT).register(entries -> {
+	entries.addAfter(Items.NETHERITE_BOOTS, EXAMPLE_HELMET, EXAMPLE_CHESTPLATE, EXAMPLE_LEGGINGS, EXAMPLE_BOOTS);
+});
+```
+
+## Adding textures for the armor
+
+Now that we have the armor registered, the only things missing are the textures and translations.
+For armors, you have to provide two textures. One for the items themselves, and one for the armor that is rendered on the player model.
+
+<!-- TODO: the desectionization fails if nested like this. See svelte.config.json -->
+<!-- ### Adding the texture to the Item-->
+
+---
+
+Adding the texture to the item is the same as any normal item: You need to add a model and a texture. For example, here is the model file for an example helmet:
+
+```json
+{
+	"parent": "item/generated",
+	"textures": {
+		"layer0": "example_mod:item/example_item_helmet"
+	}
+}
+```
+
+Repeat this for all your armor's items, replacing mod id and item name, and add the corresponding textures.
+
+<!-- TODO: the desectionization fails if nested like this. See svelte.config.json -->
+<!-- ### Adding the texture for the player model -->
+
+---
+
+Now the armor should look right in the inventory, but when you put it on, it will still show the missing texture. For that one to work we also need to add the armor model. The armor model basically consists of two textures that are organized like a player skin. However, different armor parts only draw different parts of that texture.
+Shoes, Helmet, and Chestplate use the `layer_1` texture (and have a lot more distance from the player skin itself) and Leggings use the `layer_2` texture.
+If you want, you can use this example texture as a template. The different armor parts use different tones to try help differentiating them.
+
+`assets/minecraft/models/armor/example_item_layer_1.png`:  
```

Comment:
I need to double check, but you might be able to move these out of the `minecraft` directory. If you can't i would mention that it needs to be in the `minecraft` namespace or the textures will not be found.